WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostPreviewFragment.java

```diff
@@ -21,6 +21,7 @@
     private EditPostActivity mActivity;
```

Comment:
Should I create an issue for this TODO? Otherwise I don't have high hopes of it being settled.